debian/patches/0060-add-simd-optimized-tonemapx-filter.patch

```diff
@@ -1263,1442 +1700,2454 @@ Index: FFmpeg/libavfilter/aarch64/vf_tonemapx_intrin_neon.c
++            }
++            break;
++    }
++
++    out = ff_get_video_buffer(outlink, outlink->w, outlink->h);
++    if (!out) {
++        av_frame_free(&in);
++        return AVERROR(ENOMEM);
++    }
++
++    if ((ret = av_frame_copy_props(out, in)) < 0)
++        goto fail;
++
++    /* read peak from side data if not passed in */
++    if (!peak) {
++        peak = ff_determine_signal_peak(in);
++        av_log(s, AV_LOG_DEBUG, "Computed signal peak: %f\n", peak);
++    }
++
++    out->color_trc = s->trc;
++    out->colorspace = s->spc;
++    out->color_primaries = s->pri;
++    out->color_range = s->range;
++
++    if (in->color_trc == AVCOL_TRC_UNSPECIFIED)
++        in->color_trc = AVCOL_TRC_SMPTE2084;
++    if (out->color_trc == AVCOL_TRC_UNSPECIFIED)
++        out->color_trc = AVCOL_TRC_BT709;
++
++    if (in->colorspace == AVCOL_SPC_UNSPECIFIED)
++        in->colorspace = AVCOL_SPC_BT2020_NCL;
++    if (out->colorspace == AVCOL_SPC_UNSPECIFIED)
++        out->colorspace = AVCOL_SPC_BT709;
++
++    if (in->color_primaries == AVCOL_PRI_UNSPECIFIED)
++        in->color_primaries = AVCOL_PRI_BT2020;
++    if (out->color_primaries == AVCOL_PRI_UNSPECIFIED)
++        out->color_primaries = AVCOL_PRI_BT709;
++
++    if (in->color_range == AVCOL_RANGE_UNSPECIFIED)
++        in->color_range = AVCOL_RANGE_MPEG;
++    if (out->color_range == AVCOL_RANGE_UNSPECIFIED)
++        out->color_range = AVCOL_RANGE_MPEG;
++
++    if (!s->lin_lut || !s->delin_lut) {
++        if ((ret = compute_trc_luts(s, in->color_trc, out->color_trc)) < 0)
++            goto fail;
++    }
++
++    if (!s->tonemap_lut || s->lut_peak != peak) {
++        s->lut_peak = peak;
++        if ((ret = compute_tonemap_lut(s, out->color_trc)) < 0)
++            goto fail;
++    }
++
++    coeffs = av_csp_luma_coeffs_from_avcsp(in->colorspace);
++    if (s->coeffs != coeffs) {
++        s->coeffs = coeffs;
++        s->ocoeffs = av_csp_luma_coeffs_from_avcsp(out->colorspace);
++        if ((ret = compute_yuv_coeffs(s, coeffs, s->ocoeffs, desc, odesc,
++             in->color_range, out->color_range)) < 0)
++            goto fail;
++        if ((ret = compute_rgb_coeffs(s, in->color_primaries, out->color_primaries)) < 0)
++            goto fail;
++    }
++
++    if (s->apply_dovi) {
++        AVFrameSideData *dovi_sd = av_frame_get_side_data(in, AV_FRAME_DATA_DOVI_METADATA);
++        if (dovi_sd) {
++            const AVDOVIMetadata *metadata = (AVDOVIMetadata *) dovi_sd->data;
++            const AVDOVIRpuDataHeader *rpu = av_dovi_get_header(metadata);
++            // only map dovi rpus that don't require an EL and has rpu profile == 0
++            // for performance reason we only want to do reshaping when absolutely needed
++            // such videos usually have vdr_rpu_profile == 0, for example profile 5 videos
++            // this could be wrong as there is no public documentation on this field
++            if (rpu->disable_residual_flag && rpu->vdr_rpu_profile == 0) {
```

Comment:
`rpu->vdr_rpu_profile == 0` This also seems to work for dav1 profile10.

Reply:
We don't have official support for profile10 on server side yet though.